packages/docs/pages/index.vue

```diff
@@ -73,6 +74,10 @@ export default class Index extends Vue {
     }
   }
 
+  get navigationItems () {
```

Comment:
Please use consistent naming. `Routes` should not suddenly become `NavigationItems`.